esque/controller/topic_controller.py

```diff
@@ -119,23 +130,34 @@ def get_offsets_closest_to_timestamp(
             topic_partition.partition: topic_partition.offset for topic_partition in topic_partitions_with_new_offsets
         }
 
-    def update_from_cluster(self, topic: Topic, *, retrieve_last_timestamp: bool = False) -> Topic:
+    def update_from_cluster(
+        self, topic: Topic, *, retrieve_last_timestamp: bool = False, retrieve_partition_data: bool = True
+    ) -> Topic:
         """Takes a topic and, based on its name, updates all attributes from the cluster"""
 
-        topic.partition_data = self._get_partition_data(topic, retrieve_last_timestamp)
+        if retrieve_partition_data:
+            topic.partition_data = self._get_partitions(topic, retrieve_last_timestamp)
+        else:
+            topic.partition_data = self._get_partitions(topic, retrieve_last_timestamp, get_partition_data=False)
         topic.config = self.cluster.retrieve_config(ConfigResource.Type.TOPIC, topic.name)
 
         topic.is_only_local = False
 
         return topic
 
-    def _get_partition_data(self, topic: Topic, retrieve_last_timestamp: bool) -> List[Partition]:
-
+    def _get_partitions(
+        self, topic: Topic, retrieve_last_timestamp: bool, get_partition_data: bool = True
+    ) -> List[Partition]:
         config = Config.get_instance().create_confluent_config()
         config.update({"group.id": ESQUE_GROUP_ID, "topic.metadata.refresh.interval.ms": "250"})
         with closing(confluent_kafka.Consumer(config)) as consumer:
             confluent_topic = consumer.list_topics(topic=topic.name).topics[topic.name]
             partitions: List[Partition] = []
+            if not get_partition_data:
+                return [
+                    Partition(partition_id, 0, 0, meta.isrs, meta.leader, meta.replicas, None)
```

Comment:
```suggestion
                    Partition(partition_id, -1, -1, meta.isrs, meta.leader, meta.replicas, None)
```
I think this will make it more clear, that those aren't actual offsets.